[bug] Window grows until filling screen then app crashes

### What happend?

Title?

### Code example

<details><summary><code>main.go</code></summary>

```golang
package main

import (
	g "github.com/AllenDang/giu"
)

func loop() {
	g.Window("main").Layout(
		g.Labelf("hi"),
	)
}

func main() {
	wnd := g.NewMasterWindow("main", 400, 200, 0)
	wnd.Run(loop)
}

```

</details>


### To Reproduce

1. Run my code
2. see the crash...


### Version

(latest)

### OS

Windows 10

Answers:
- It does not matter, it's the same behavior if the only content is InputTextMultiline as well.

Message and call stack: 
```
PS C:\Temp\giu> go run "c:\Temp\giu\examples\helloworld\helloworld.go"
panic: Assertion failed!
File: D:/a/cimgui-go/cimgui-go/cimgui/imgui/imgui.cpp, Line 10226

Expression: g.Style.WindowMinSize.x >= 1.0f && g.Style.WindowMinSize.y >= 1.0f && "Invalid style setting."


goroutine 1 [running]:
github.com/AllenDang/cimgui-go.init.func1({0xc0001c1500?, 0x0?}, {0xc00000ed20?, 0x7ff7bc6a3ac0?}, 0xc000138000?)
        C:/Users/Alex/go/pkg/mod/github.com/!allen!dang/cimgui-go@v0.0.0-20240424153022-294abe7370df/assert.go:28 +0x65
github.com/AllenDang/cimgui-go.goImguiAssertHandler(0xc00007fc50?, 0x7ff7bce316c0, 0xbc6296ef?)
        C:/Users/Alex/go/pkg/mod/github.com/!allen!dang/cimgui-go@v0.0.0-20240424153022-294abe7370df/assert.go:45 +0x67
github.com/AllenDang/cimgui-go._Cfunc_igGLFWRunLoop(0x25ece753cf0, 0x7ff7bc6a58e0, 0x7ff7bc6a5920, 0x7ff7bc6a5960, 0x7ff7bc6a59e0)
        _cgo_gotypes.go:24237 +0x4d
github.com/AllenDang/cimgui-go.(*GLFWBackend).Run.func1(0x7ff7bc55da01?)
        C:/Users/Alex/go/pkg/mod/github.com/!allen!dang/cimgui-go@v0.0.0-20240424153022-294abe7370df/glfw_backend.go:249 +0x9c    
github.com/AllenDang/cimgui-go.(*GLFWBackend).Run(0xc00006c050?, 0xc00006c050?)
        C:/Users/Alex/go/pkg/mod/github.com/!allen!dang/cimgui-go@v0.0.0-20240424153022-294abe7370df/glfw_backend.go:249 +0x32    
main.main.(*MasterWindow).Run.func1()
        C:/Temp/giu/MasterWindow.go:244 +0xcb
github.com/AllenDang/giu.mainthreadCallPlatform(...)
        C:/Temp/giu/mainthread_windows.go:12
github.com/AllenDang/giu.(*MasterWindow).Run(...)
        C:/Temp/giu/MasterWindow.go:236
main.main()
        c:/Temp/giu/examples/helloworld/helloworld.go:18 +0x67
exit status 2
```

--- a/MasterWindow.go
+++ b/MasterWindow.go
@@ -116,16 +116,16 @@ func NewMasterWindow(title string, width, height int, flags MasterWindowFlags) *
 
 	mw.SetBgColor(colornames.Black)
 
-	return mw
-}
-
-func (w *MasterWindow) setTheme() (fin func()) {
 	// Scale DPI in windows
 	if runtime.GOOS == "windows" {
 		xScale, _ := Context.backend.ContentScale()
 		imgui.CurrentStyle().ScaleAllSizes(xScale)
 	}
 
+	return mw
+}
+
+func (w *MasterWindow) setTheme() (fin func()) {
 	imgui.PushStyleVarFloat(imgui.StyleVarWindowRounding, 2)
 	imgui.PushStyleVarFloat(imgui.StyleVarFrameRounding, 4)
 	imgui.PushStyleVarFloat(imgui.StyleVarGrabRounding, 4)

